3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -124,7 +124,7 @@ function add_member() {
     return 1
   fi
 
-  info "Successfully joined replica set"
+  info "Successfully connected to replica set"
```

Comment:
I think this was a non-improvement. To "connect to" a replica set is something like `mongo --host rs0/host1,host2,...`, and that is not what this message is about. IMHO "joined" seemed more appropriate.